### PR TITLE
chore(TBD-10493): Move our master to maintenance/7.3

### PR DESCRIPTION
**Pull request by lbourgeois**
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

File: i18n/plugins/org.talend.demo.bigdata.ce.nl/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -2,5 +2,5 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: Nl Fragment
 Bundle-SymbolicName: org.talend.demo.bigdata.ce.nl
-Bundle-Version: 7.3.1.qualifier
+Bundle-Version: 7.4.1.qualifier
```
Comment (CarlosTalend) [inline i18n/plugins/org.talend.demo.bigdata.ce.nl/META-INF/MANIFEST.MF:5]: Must be 7.3. Reverting 508213bfc309392916499885ee517c79f4f2fde8 normally fixes the automatic modifications

Comment (lbourgeois) [reply i18n/plugins/org.talend.demo.bigdata.ce.nl/META-INF/MANIFEST.MF]: Reverted to 7.3.1

File: main/plugins/org.talend.hadoop.distribution.cdh5x/resources/builtin/cdhx/Cloudera_CDH6_1_1.json [hunk 2]
```diff
@@ -9070,6 +9074,10 @@
         "childNodes" : [ ],
         "id" : "DYNAMIC_Cloudera_CDH6_1_1_hadoop_yarn_server_web_proxy_3_0_0_cdh6_1_1_jar",
         "tagName" : "library"
+      }, {
+        "childNodes" : [ ],
+        "id" : "DYNAMIC_Cloudera_CDH6_1_1_hive_exec_1_2_1_talend",
```
Comment (CarlosTalend) [inline main/plugins/org.talend.hadoop.distribution.cdh5x/resources/builtin/cdhx/Cloudera_CDH6_1_1.json:9079]: I see another hive exec (1_1 a bit below). I wonder if there could be a conflict

File: main/plugins/org.talend.hadoop.distribution.dbr350/src/main/java/org/talend/hadoop/distribution/dbr350/DBR350Distribution.java [hunk 3]
```diff
@@ -29,15 +29,14 @@
...
-import org.talend.hadoop.distribution.constants.databricks.IDatabricksDistribution;
 import org.talend.hadoop.distribution.dbr350.modulegroup.DBR350HiveOnSparkModuleGroup;
 import org.talend.hadoop.distribution.dbr350.modulegroup.DBR350SparkBatchModuleGroup;
 import org.talend.hadoop.distribution.dbr350.modulegroup.DBR350SparkStreamingModuleGroup;
 import org.talend.hadoop.distribution.dbr350.modulegroup.node.sparkbatch.DBR350SparkBatchAzureNodeModuleGroup;
 import org.talend.hadoop.distribution.dbr350.modulegroup.node.sparkstreaming.DBR350SparkStreamingKinesisNodeModuleGroup;
 
 public class DBR350Distribution extends AbstractDatabricksDistribution implements SparkBatchComponent,
-        SparkStreamingComponent, HiveOnSparkComponent, IDatabricksDistribution {
+        SparkStreamingComponent, HiveOnSparkComponent {
```
Comment (CarlosTalend) [inline main/plugins/org.talend.hadoop.distribution.dbr350/src/main/java/org/talend/hadoop/distribution/dbr350/DBR350Distribution.java:39]: not sure if this removal should be done

File: main/plugins/org.talend.hadoop.distribution.dbr550/src/main/java/org/talend/hadoop/distribution/dbr550/DBR550Distribution.java [hunk 4]
```diff
@@ -29,15 +29,14 @@
...
-import org.talend.hadoop.distribution.constants.databricks.IDatabricksDistribution;
 import org.talend.hadoop.distribution.dbr550.modulegroup.DBR550HiveOnSparkModuleGroup;
 import org.talend.hadoop.distribution.dbr550.modulegroup.DBR550SparkBatchModuleGroup;
 import org.talend.hadoop.distribution.dbr550.modulegroup.DBR550SparkStreamingModuleGroup;
 import org.talend.hadoop.distribution.dbr550.modulegroup.node.sparkbatch.DBR550SparkBatchAzureNodeModuleGroup;
 import org.talend.hadoop.distribution.dbr550.modulegroup.node.sparkstreaming.DBR550SparkStreamingKinesisNodeModuleGroup;
 
 public class DBR550Distribution extends AbstractDatabricksDistribution implements SparkBatchComponent,
-        SparkStreamingComponent, HiveOnSparkComponent, IDatabricksDistribution {
+        SparkStreamingComponent, HiveOnSparkComponent {
```
Comment (CarlosTalend) [inline main/plugins/org.talend.hadoop.distribution.dbr550/src/main/java/org/talend/hadoop/distribution/dbr550/DBR550Distribution.java:39]: ditto for the removal

File: main/plugins/org.talend.hadoop.distribution/src/main/java/org/talend/hadoop/distribution/AbstractDatabricksDistribution.java [hunk 5]
```diff
@@ -12,12 +12,22 @@
...
+public abstract class AbstractDatabricksDistribution extends AbstractDistribution implements IDatabricksDistribution {
 
     /**
      * Get the spark version to be passed in rest call for transient cluster creation
      */
     public String getTransientClusterSparkVersion() {
         return "3.5.x-scala2.11";
     };
+
+    public List<EDatabriksCloudProvider> getSupportCloudProviders() {
```
Comment (CarlosTalend) [inline main/plugins/org.talend.hadoop.distribution/src/main/java/org/talend/hadoop/distribution/AbstractDatabricksDistribution.java:30]: not sure if used correctly elsewhere, but Databricks is misspelled here (and on the line below)

File: main/plugins/org.talend.hadoop.distribution/src/main/java/org/talend/hadoop/distribution/constants/databricks/EDatabriksCloudProvider.java [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+//
+// You should have received a copy of the agreement
+// along with this program; if not, write to Talend SA
+// 9 rue Pages 92150 Suresnes, France
+//
+// ============================================================================
+package org.talend.hadoop.distribution.constants.databricks;
+
+import org.talend.hadoop.distribution.i18n.Messages;
+public enum EDatabriksCloudProvider {
```
Comment (CarlosTalend) [inline main/plugins/org.talend.hadoop.distribution/src/main/java/org/talend/hadoop/distribution/constants/databricks/EDatabriksCloudProvider.java:16]: Also faulty, but it looks like the naming is consistent everywhere

File: main/plugins/org.talend.hadoop.distribution/src/main/java/org/talend/hadoop/distribution/constants/databricks/IDatabricksDistribution.java [hunk 7]
```diff
@@ -12,10 +12,15 @@
...
+import java.util.List;
+
 @SuppressWarnings("nls")
 public interface IDatabricksDistribution {
 
     static final String DISTRIBUTION_NAME = "DATABRICKS";
 
     static final String DISTRIBUTION_DISPLAY_NAME = "Databricks";
+
+    List<EDatabriksCloudProvider> getSupportCloudProviders();
```
Comment (CarlosTalend) [inline main/plugins/org.talend.hadoop.distribution/src/main/java/org/talend/hadoop/distribution/constants/databricks/IDatabricksDistribution.java:24]: still consistent

File: main/plugins/org.talend.repository.hadoopcluster/src/org/talend/repository/hadoopcluster/ui/DataBricksInfoForm.java [hunk 8]
```diff
@@ -28,10 +30,13 @@
 import org.eclipse.swt.widgets.Composite;
 import org.eclipse.swt.widgets.Group;
 import org.talend.commons.ui.swt.formtools.Form;
+import org.talend.commons.ui.swt.formtools.LabelledCombo;
 import org.talend.commons.ui.swt.formtools.LabelledText;
 import org.talend.core.database.conn.ConnParameterKeys;
 import org.talend.core.hadoop.repository.HadoopRepositoryUtil;
 import org.talend.core.model.properties.ConnectionItem;
+import org.talend.hadoop.distribution.constants.databricks.EDatabriksCloudProvider;
```
Comment (CarlosTalend) [inline main/plugins/org.talend.repository.hadoopcluster/src/org/talend/repository/hadoopcluster/ui/DataBricksInfoForm.java:38]: consistent

Comment (lbourgeois) [reply main/plugins/org.talend.repository.hadoopcluster/src/org/talend/repository/hadoopcluster/ui/DataBricksInfoForm.java:38]: Agree should be renamed with correct spelling in separate ticket

File: main/plugins/org.talend.hadoop.distribution.dbr350/src/main/java/org/talend/hadoop/distribution/dbr350/DBR350Distribution.java (hunk 3 above)
Comment (lbourgeois) [reply main/plugins/org.talend.hadoop.distribution.dbr350/src/main/java/org/talend/hadoop/distribution/dbr350/DBR350Distribution.java:39]: Will check from where it is coming

Comment (lbourgeois) [reply main/plugins/org.talend.hadoop.distribution.dbr350/src/main/java/org/talend/hadoop/distribution/dbr350/DBR350Distribution.java:39]: Move to https://github.com/Talend/tbd-studio-se/blob/0cc7b05827873bb534a14a89b9dca187cd02f4a6/main/plugins/org.talend.hadoop.distribution/src/main/java/org/talend/hadoop/distribution/AbstractDatabricksDistribution.java#L21 in scope of https://github.com/Talend/tbd-studio-se/commit/0cc7b05827873bb534a14a89b9dca187cd02f4a6

File: main/plugins/org.talend.hadoop.distribution.cdh5x/resources/builtin/cdhx/Cloudera_CDH6_1_1.json (hunk 2 above)
Comment (lbourgeois) [reply main/plugins/org.talend.hadoop.distribution.cdh5x/resources/builtin/cdhx/Cloudera_CDH6_1_1.json:9079]: Agree, looks strange, I will overwrite the built-in with master version

**Pull request merged**